Cargo.toml

```diff
@@ -12,6 +12,7 @@ categories = ["embedded", "no-std", "data-structures"]
 
 [features]
 std = []
+nightly = []
```

Comment:
I have not published a crate that exposes an optional nightly feature before.

What are the trade-offs of calling this "nightly" vs "nightly_const_convert"/"nightly_const_trait_impl"?

Reply:
I suppose if someone just wants one of them, they could file an issue and we could still do `nightly = [nightly_const_convert, nightly_const_trait_impl]`.
I remember some project did that, but not which one.

Besides that, if we had those separated, if one of them becomes stable we could... not do anything with that info either. Maybe someone really only needs Number to be a const trait, but we could provide that if someone asks. The other one, const_convert, needs const trait iirc.

If we add more nightly functionality, it's either needed, so we add it to the nightly feature, or special, which would need a different feature.

Reply:
Ok, found a funny way to contact you, here, in this comment I will delete right after:
are you on discord or sth? I'm only on discord and... mail.